packages/core/src/utils/get-paths.ts

```diff
@@ -28,15 +29,23 @@ export function getFilePaths(
   let mergedPathsToIgnore = [...excludePaths, ...PATHS_TO_IGNORE];
 
   if (globs.length > 0) {
-    return expandFileGlobs(globs, basePath, mergedPathsToIgnore);
+    debug('Expanding file globs...');
+    let expandedGlobs = expandFileGlobs(globs, basePath, mergedPathsToIgnore);
+    debug(`${expandedGlobs.length} file paths fetched!`);
+
+    return expandedGlobs;
   }
 
+  debug('Fetching all file paths via walkSync...');
```

Comment:
```suggestion
  debug('start finding all file paths');
```

We should avoid leaking implementation details into these messages, such as `walkSync`.